paper-server/src/main/java/io/papermc/paper/registry/data/util/Checks.java

```diff
@@ -40,6 +40,13 @@ public static int asArgumentMin(final int value, final String field, final int m
         return value;
     }
 
+    public static float asArgumentMinExclusive(final float value, final String field, final float min) {
+        if (value <= min) {
+            throw new IllegalArgumentException("argument " + field + " must be [" + min + ",+inf)");
```

Comment:
This message should be different to show that the minimum must be exclusive ( or ]